randovania/game_description/world/area.py

```diff
@@ -23,19 +23,29 @@ def __repr__(self):
         return "Area[{}]".format(self.name)
 
     def __hash__(self):
-        return self.area_asset_id
+        return hash(self.name)
+
+    @property
+    def in_dark_aether(self) -> Optional[bool]:
+        return self.extra.get("in_dark_aether")
```

Comment:
rather than being optional, would it make more sense to do `return self.extra.get("in_dark_aether', False)`?

Reply:
I was thinking of removing this getter, but I can't see a way to decently do this for a while, sadly.